README.md

```diff
@@ -29,6 +28,7 @@ To get a local copy up and running follow these simple example steps.
 ### Setup
 
 ### Install
+- can sampily clone the project to local environment and run it.
 
 ### Usage
 
```

Comment:
- [ ] To make your README look professional, please delete the unnecessary headings and default template text. 
<img width="853" alt="Screenshot 2022-08-31 at 11 13 54" src="https://user-images.githubusercontent.com/77038610/187628423-6d287d61-2f13-4fe1-92e3-450213e22c0b.png">
<img width="406" alt="Screenshot 2022-08-31 at 11 14 50" src="https://user-images.githubusercontent.com/77038610/187628707-076bac06-504c-49ea-b093-14253e7a05e5.png">